EIPS/eip-5850.md

```diff
@@ -0,0 +1,71 @@
+---
+eip: 5850
+title: Complex Numbers stored in Bytes32 types
+description: Store real and imaginary parts of complex numbers in the lower 16 bytes and upper 16 bytes respectively of a bytes32 type.
```

Comment:
This should use most significant / least significant bytes, as those are the correct terms

Reply:
Agreed

Reply:
Agree EIP-5850